File: Assets/Custom Assets/Scripts/Heal.js
#pragma strict
private var receptor:PlayerStatsHandler;

public var heal_amount:int=500;

function Start () {

}

function Update () {

}

function OnTriggerEnter(other : Collider) {
		if(other.gameObject.CompareTag("Player")){ //HELEA AL JUGADOR
			receptor = other.GetComponent(PlayerStatsHandler);
			receptor.heal(heal_amount);
			Destroy(gameObject);
		}
	}

File: Assets/Custom Assets/Scripts/Heal.cs
using UnityEngine;

public class Heal : MonoBehaviour
{
    private PlayerStatsHandler receptor;
    public int heal_amount = 500;

    void Start()
    {
    }

    void Update()
    {
    }

    void OnTriggerEnter(Collider other)
    {
        if (other.gameObject.CompareTag("Player"))
        {
            // Heal the player
            receptor = other.GetComponent<PlayerStatsHandler>();
            receptor.Heal(heal_amount);
            Destroy(gameObject);
        }
    }
}